.zuul.yaml

```diff
@@ -8,8 +8,6 @@
     gate:
       jobs:
         - golang-make-functional
-        - golang-make-vet
-        - goreleaser-build
```

Comment:
those are "duplicate", but they are executed in another step of the CI. They are there to verify, that when we are merging the change (and not when we first opened PR perhaps some days or weeks before) - it is still working. This helps to catch problems in the dependent projects or 3rd party changes breaking the build.
Removing verification that the build works - a clear **no go**. This can lead to the inability to build release later.
General regulation: gate is even more strict with jobs compared to check and not vice versa, since it verifies that "master" is always usable

Reply:
Is not `check` run every pushed commit? How can it not fail if something is broken?